src/forms/stock-dispense/stock-dispense.component.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ComboBox, InlineLoading, InlineNotification, Layer } from '@carbon/react';
+import { type MedicationDispense, type InventoryItem } from '../../types';
+import { useDispenseStock } from './useDispenseStock';
+import { formatDate } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+
+type StockDispenseProps = {
+  medicationDispense: MedicationDispense;
+  updateInventoryItem: (inventoryItem: InventoryItem) => void;
+  inventoryItem: InventoryItem;
+};
+
+const StockDispense: React.FC<StockDispenseProps> = ({ medicationDispense, updateInventoryItem }) => {
+  const { t } = useTranslation();
+  const drugUuid = medicationDispense?.medicationReference?.reference?.split('/')[1];
+  const { inventoryItems, error, isLoading } = useDispenseStock(drugUuid);
+
+  const toStockDispense = (inventoryItems) => {
+    return t(
+      'stockDispenseDetails',
+      'Batch: {{batchNumber}} - Quantity: {{quantity}} ({{quantityUoM}}) - Expiry: {{expiration}}',
+      {
+        batchNumber: inventoryItems.batchNumber,
+        quantity: Math.floor(inventoryItems.quantity),
+        quantityUoM: inventoryItems.quantityUoM,
+        expiration: formatDate(new Date(inventoryItems.expiration)),
```

Comment:
format this to MM/YYYY

Reply:
Any specific reason why it should marked that way?

Reply:
Tend to agree with @ibacher